tyr/servers/server.py

```diff
@@ -371,21 +386,58 @@ def blockdevicemapping(self):
 
         return bdm
 
+    def get_subnet_vpc_id(self, subnet_id):
+        vpcConn = VPCConnection()
```

Comment:
Python recommends using

> lowercase with words separated by underscores as necessary to improve readability

for variable names as opposed to mixed case.

([PEP 008 - Method Names and Instance Variables][0])

[0]: https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables